### DISCUSSION TITLE
How to fix scrollBar after resize?

### DISCUSSION THREAD

**Discussion by opadfnezig** [tui-cs/Terminal.Gui#1109] [Q&A] [answered]
So, I'm new to the community, started 2-3 days ago and have a lot of questions.
The most crucial thing for me right now, is how to deal with weird resize issues? It seems like a bug, but not shure if it is, have I missed something? I've coppied a code from title page of the repo. I run the project, resize it to fullscreen, than resize back and.. 
![image](https://user-images.githubusercontent.com/38683305/107713720-8aa70400-6cd4-11eb-8c0a-131d39724bd2.png) 
terrible scrollBar appears. It's fixed in demo(in sources), but actually, it's the only thing which works for me in demo(slow but works), so don't think that to take a code from it is a great idea. I think I'll try in on the weekends, but if you have any answers/solutions, u're welcome.

**Comment by BDisp** [comment, marked as answer]
The `demo.cs` is slow because it is using `Application.UseSystemConsole` and thus using `NetDriver`. If you comment that line it will use `WindowsDriver` if you are running on a `Windows 10` or will use `NCurses` if you are running `Linux`.
If you want to avoid the scrollbar from appearing, you must disable `Height as Buffer` on the `Diagnostic` menu. However, there is a bug that preventing the size not fit correctly on the console. A possible fix is on PR #1066.

**Reply by opadfnezig** [reply]
Ok, I've found this field, but the problem is kinda funny. I can see it in demo, I've found source code there and can see `public static bool HeightAsBuffer`, but in my project it doesn't exists for some reasons.. here is definition of `Application`:
![image](https://user-images.githubusercontent.com/38683305/107752628-b0112d80-6d27-11eb-8b1a-1078de840d7d.png)
And as a result, vs says me that it doesn't exsists... 
![image](https://user-images.githubusercontent.com/38683305/107752701-cae3a200-6d27-11eb-9dc0-53350f6dbc2d.png)
So, is it just me, or no?

**Reply by BDisp** [reply]
That not exist in the nuget package yet, only in the master version.

**Reply by opadfnezig** [reply]
Thank you for the explanation. I will wait for update, I guess. I can not resize, and maybe I will just disable it for now.
